docs/tutorials/pasqal/getting_started.ipynb

```diff
@@ -75,7 +75,8 @@
     "except ImportError:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    too_far_away_circuit = cirq.Circuit()
</pre><p>What's the need for this circuit? It seems it is never used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5316/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Removed.  Was a remnant of when the below operation was a circuit.